app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h3>This is <%= @project.name %>'s project </h3>
+<p>Materials: <%= @project.material %> </p>
+<p>Theme: <%= @project.challenge.theme %> </p>
+<p>Count of Contestants: <%= @project.contestants.count %></p>
```

Comment:
Good work recognizing that this is an MVC infraction. The rubric allows for one infraction so it's okay from an evaluation standpoint but we would want to see you encapsulate this into a model method, and then also test that model method